browser/profiles/brave_profile_manager.cc

```diff
@@ -61,12 +59,12 @@ BraveProfileManager::BraveProfileManager(const base::FilePath& user_data_dir)
     : ProfileManager(user_data_dir) {
   MigrateProfileNames();
 
-  registrar_.Add(this, chrome::NOTIFICATION_PROFILE_CREATED,
-                 content::NotificationService::AllSources());
+  AddObserver(this);
 }
 
 BraveProfileManager::~BraveProfileManager() {
   std::vector<Profile*> profiles = GetLoadedProfiles();
+  RemoveObserver(this);
```

Comment:
super small nit, but maybe this shouldn't be injected between obtaining and using of `profiles`

Reply:
Fixed in https://github.com/brave/brave-core/commit/6ac54a693256692567f61fbb695e0f89fdc36620